README.md

```diff
@@ -210,11 +210,11 @@ Name | Description
 
 The [VMware REST modules guide](https://docs.ansible.com/ansible/devel/scenario_guides/guide_vmware_rest.html) gives a step by step introduction of the collection.
 
-## Testing and Development
```

Comment:
Please keep the same section name or make Testing with `ansible-test` a section rather than a subsection